docs/performance_tuning.md

```diff
@@ -29,6 +29,25 @@ For example with ~50 clients sending the requests to the server with 48 cores, s
 `--plugin_config '{"CPU_THROUGHPUT_STREAMS": "24"}'`
 
 
+## Performance Hints
+The `PERFORMANCE_HINT` plugin config property enables you to specify a performance mode for the plugin to be more efficient for particular use cases.
+
+#### THROUGHPUT
+This mode prioritizes high throughput, balancing between latency and power. It is best suited for tasks involving multiple jobs, like inference of video feeds or large numbers of images.
+
+#### LATENCY
+This mode prioritizes low latency, providing short response time for each inference job. It performs best for tasks where inference is required for a single input image, like a medical analysis of an ultrasound scan image. It also fits the tasks of real-time or nearly real-time applications, such as an industrial robot's response to actions in its environment or obstacle avoidance for autonomous vehicles.
+Note that currently the `PERFORMANCE_HINT` property is supported by CPU and GPU devices only. [More information](https://github.com/openvinotoolkit/openvino/blob/928076ed319fcf172d24d1af4c6844e39c1bc100/docs/OV_Runtime_UG/auto_device_selection.md).
+
+To enable Performance Hints for your application, use the following command:
+```
+docker run --rm -d -v <model_path>:/opt/model -p 9001:9001 openvino/model_server:latest \
+--model_path /opt/model --model_name my_model --port 9001 \
+--plugin_config '{"PERFORMANCE_HINT": "THROUGHTPUT"}'
```

Comment:
Could we include `target_device` parameter here?

Reply:
Which target device in particular?

Reply:
Maybe both? We would have exemplary commands for both users of CPU and GPU.

Reply:
I don't think we should do that, we would be ignoring other devices like AUTO/MULTI/HETERO/HDDL/MYRIAD